### ISSUE TITLE
Is this ready for use? 

### ISSUE THREAD

**Issue by Jamongkad** [ericcastro/ECStretchableHeaderView#1]
Quick question is this ready for use? I can't seem to get this code working. I can a blank white box and seeing my content able to stretch...but I don't see the box.

self.headerView = [[ECStretchableHeaderView alloc] initWithFrame:CGRectMake(0, 0, 200, 200)];
[self.headerView attachToScrollView:self.scrollView inset:0.0];
[self.headerView setBackgroundColor:[UIColor redColor]];

**Comment by ericcastro** [comment]
hey, I'm sorry - I originally built this control for an app I worked on, and continued developing it there, but since I didn't see much interest from people on using it (until now ;P) I completely forgot to update it here.

I will get the code updated soon and will check your code and also provide a working example.

Thanks

**Comment by Jamongkad** [comment]
Great! I'm trying to attach controls to it as well. Perhaps a detailed tutorial on how to use it?

**Comment by Jamongkad** [comment]
Hi Eric any news on this? I'm sorry if I seem petulant but this is something I need very soon.

**Comment by ericcastro** [comment]
Hi!
I have now updated the project with the latest code and an example for you to try; the main thing to know is that the ECStretchableHeaderView relies on a height constraint on it to function; therefore your layout must use AutoLayout and have this constraint assigned to the .heightConstraint property before calling the attachToScrollView method.

See the example I provided for more info, otherwise just ask me here.

I plan on improving the documentation soon, but this should get you started.
